packages/lime-system/files/usr/lib/lua/lime/config.lua

```diff
@@ -198,8 +189,17 @@ function config.uci_commit_all()
 end
 
 function config.main()
-	config.sanitize()
-	config.sanitize(config.UCI_NODE_NAME)
+	config.initialize_config_file(config.UCI_AUTOGEN_NAME)
```

Comment:
Why are you initializing lime-autogen if a few lines after config.uci_autogen() will empty it?

Reply:
Right!, just because I didn't read all the code I only changed what it was needed. Will fix